### PR TITLE
Fix version

### PR DESCRIPTION
**Pull request by jrmlhermitte**

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
this fixes doc issue temporarily....

File: doc/source/tutorial.rst [hunk 1]
```diff
@@ -15,7 +15,10 @@ concurrent "streams" of data in an organized way.
 
     import os
     import yaml
+    import epics
+    epics.__version__ = "0.0.0"
     from databroker import temp_config
+    from bluesky.examples import det, motor
```
Comment (tacaswell) [inline doc/source/tutorial.rst]: This should be `from ophyd.sim`

Comment (jrmlhermitte) [reply doc/source/tutorial.rst]: thanks

**Comment by tacaswell** [comment]
The `_as_gen` files should not be under source control.

**Comment by danielballan** [comment]
I rebased to remove generated files from PR. I discovered that `.gitignore` thought these files were under the directory `generated/` but in fact they were going in to `_as_gen/`. In the docs, I updated the autosummary directives to output to `generated/` which is consistent with our other projects.

**Comment by danielballan** [comment]
Thanks for kicking this off @jrmlhermitte!

**Comment by jrmlhermitte** [comment]
thanks @danielballan !

**Pull request merged**